src/errors.rs

```diff
@@ -61,4 +61,6 @@ pub enum Error {
     NoCloid,
     #[error("ECDSA signature failed: {0:?}")]
     SignatureFailure(String),
+    #[error("Vault address not provided")]
```

Comment:
nit: align error message with name for consistency, either `VaultAddressNotProvided` and `"Vault address not provided"` or `VaultAddressNotFound` and `"Vault address not found"`

Reply:
Done